.jhipster/CancerType.json

```diff
@@ -52,6 +52,13 @@
       "otherEntityName": "cancerType",
       "otherEntityRelationshipName": "children",
       "relationshipName": "parent"
+    },
+    {
+      "relationshipType": "many-to-many",
+      "otherEntityName": "clinicalTrialsGovCondition",
+      "otherEntityRelationshipName": "cancerType",
+      "relationshipName": "clinicalTrialsGovCondition",
+      "ownerSide": false
```

Comment:
I'm not sure this is missed from the previous changes.

Reply:
There are few changes are caused by the generator related to `clinicalTrialsGovCondition`. Need a discussion on this.